x-pack/plugins/apm/public/components/app/correlations/context_popover/top_values.tsx

```diff
@@ -67,103 +234,66 @@ export function TopValues({ topValueStats, onAddFilter, fieldValue }: Props) {
-                              ? value.key.toString()
-                              : value.key,
-                          include: true,
-                        });
-                      }}
-                      aria-label={i18n.translate(
-                        'xpack.apm.correlations.fieldContextPopover.addFilterAriaLabel',
-                        {
-                          defaultMessage: 'Filter for {fieldName}: "{value}"',
-                          values: { fieldName, value: value.key },
-                        }
-                      )}
-                      data-test-subj={`apmFieldContextTopValuesAddFilterButton-${value.key}-${value.key}`}
-                      style={{
-                        minHeight: 'auto',
-                        width: theme.eui.euiSizeL,
-                        paddingRight: 2,
-                        paddingLeft: 2,
-                        paddingTop: 0,
-                        paddingBottom: 0,
-                      }}
-                    />
-                    <EuiButtonIcon
-                      iconSize="s"
-                      iconType="minusInCircle"
-                      onClick={() => {
-                        onAddFilter({
-                          fieldName,
-                          fieldValue:
-                            typeof value.key === 'number'
-                              ? value.key.toString()
-                              : value.key,
-                          include: false,
-                        });
-                      }}
-                      aria-label={i18n.translate(
-                        'xpack.apm.correlations.fieldContextPopover.removeFilterAriaLabel',
-                        {
-                          defaultMessage: 'Filter out {fieldName}: "{value}"',
-                          values: { fieldName, value: value.key },
-                        }
-                      )}
-                      data-test-subj={`apmFieldContextTopValuesExcludeFilterButton-${value.key}-${value.key}`}
-                      style={{
-                        minHeight: 'auto',
-                        width: theme.eui.euiSizeL,
-                        paddingTop: 0,
-                        paddingBottom: 0,
-                        paddingRight: 2,
-                        paddingLeft: 2,
-                      }}
-                    />
-                  </>
-                ) : null}
-              </EuiFlexGroup>
+              <TopValue
+                value={value}
+                barColor={barColor}
+                valueText={valueText}
+                onAddFilter={onAddFilter}
+                progressBarMax={progressBarMax}
+                isHighlighted={isHighlighted}
+                fieldName={fieldName}
+              />
             </>
           );
         })}
+
+      {Array.isArray(fieldValueStats?.topValues) && (
+        <>
+          <EuiHorizontalRule margin="s" />
+          <EuiText size="xs">
```

Comment:
```suggestion
          <EuiText size="xs" color="subdued">
```
Lowering the contrast of this message since it's just a help text label for the result.

Reply:
Or looking at the EUI 41.0.0 upgrade PR, should `color="text"` be used rather than `subdued`?

Reply:
`text` is darker contrast than `subdued` - what's the EUI upgrade change you're referring to?

<img width="1036" alt="CleanShot 2021-11-11 at 15 36 24@2x" src="https://user-images.githubusercontent.com/4104278/141316201-608d356e-9b5b-4dbe-a0c5-2a1981103759.png">

Reply:
Sorry - you're right @formgeist, the EUI 41.0.0 upgrade only changes it to `text` for `EuiButtonIcon`.